adagrams/game.py

```diff
@@ -1,13 +1,111 @@
+# comment to practice push to GitHub
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+POINTS = {
+    1:["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2:["D","G"],
+    3:["B","C","M","P"],
+    4:["F","H","V","W","Y"],
+    5:["K"],
+    8:["J","X"],
+    10:["Q","Z"]
+    }
+
 def draw_letters():
-    pass
+    total_letter_pool = []
+    hand = []
+
+    # create a list with the total amount of letters: 
+    for letter, quantity in LETTER_POOL.items():
+        total_letter_pool.extend([letter for i in range(quantity)])
+    
+    # draw 10 random letters from list of letters and append it to the hand
+    for i in range(10):
+        random_index = randint(0,len(total_letter_pool)-1)
```

Comment:
```suggestion
        random_index = randint(0, len(total_letter_pool) - 1)
```
You're missing white spaces after the comma and we should surround operators like `-` with white spaces. It's a small thing, but as you write more code (and eventually contribute to a large codebase) it's important to be consistent with whitespaces so that the whole project stays neat.

You can review the PEP8 style guide on white spaces [here](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements).